packages/next/build/swc/options.js

```diff
@@ -59,6 +62,7 @@ function getBaseSWCOptions({
       experimental: {
         keepImportAssertions: true,
         plugins,
+        cacheRoot: path.join(nextConfig?.distDir ?? '.next', 'swc'),
```

Comment:
```suggestion
        cacheRoot: path.join(nextConfig?.distDir ?? '.next', 'cache', 'swc'),
```
Should this be nested under the `cache` folder? Also can we add an assertion to an existing test to verify this folder is created in the expected location?

Reply:
Both sound correct to me, tried to update PR

1. try to calculate cache location correctly with putting swc cache under `/cache`
2. tried to add assertion to integration test.